Lib9c/Model/State/WeeklyArenaState.cs

```diff
@@ -222,7 +226,9 @@ public Address[] GetAgentAddresses(int count)
                     break;
             }
 
+#pragma warning disable LAA1002
```

Comment:
Is it okay we don't mind the order of addresses?

Reply:
I don't think it's a problem. but, I'll change it.

Reply:
i found there is no using this method. i'll delete this method

Reply:
🆗